tailwind.config.js

```diff
@@ -0,0 +1,148 @@
+module.exports = {
+	separator: '--',
+	prefix: '',
+	theme: {
+		colors: {
+			'transparent': 'transparent',
+			'black': '#000',
+			'white': '#fff',
+			'ubc-blue': '#002145',
+			// rgb - (0,33,69)
+			'ubc-blue-sea': '#003A79',
```

Comment:
May not be best to create new name for UBC colour.

Could we keep it consistent with how we differentiate them currently? Primary and Secondary colour, and just by number?
ubc-primary-1: #002145
ubc-secondary-1: #0055B7
ubc-secondary-2: #00A7E1

or ubc-brand-primary-1, ubc-brand-secondary-1, ubc-brand-secondary-2 if we want to be more explicit.

etc...

Reply:
I disagree. Working with the colours is challenging because the [names](https://brand.ubc.ca/guidelines/downloads/ubc-colour-and-fonts/) are too abstract. The difference between secondary-1, 2, 3, etc. is lost on me and really doesn't denote anything of value. 

That said, if we move to css variables for colours, more abstract is better, but I would hope I could make a reasonable guess as to how secondary-1 was different from secondary-2.

Not a hill I'm willing to die on, but I found in working with them it was MUCH easier to have a name that was reminiscent of the colour I could expect.

For now, why don't I remove the colours from the tailwind config and make this a separate discussion?